adcm_aio_client/core/objects/cm.py

```diff
@@ -297,3 +293,54 @@ async def _get_hosts_from_arg_or_filter(
             hosts = await self.filter(filters)  # type: ignore  # TODO
 
         return hosts
+
+
+class Action(InteractiveChildObject):
+    PATH_PREFIX = "actions"
+
+    def __init__(self: Self, parent: InteractiveObject, requester: Requester, data: dict[str, Any]) -> None:
+        super().__init__(parent, requester, data)
+        self._verbose = False
+
+    @cached_property
+    def name(self: Self) -> str:
+        return self._data["name"]
+
+    @cached_property
+    def display_name(self: Self) -> str:
+        return self._data["displayName"]
+
+    async def run(self: Self) -> dict:  # TODO: implement Task, return Task
+        return (await self._requester.post(*self.get_own_path(), "run", data={"isVerbose": self._verbose})).as_dict()
+
+    @async_cached_property
+    async def _mapping_rule(self: Self) -> list[dict]:
+        return (await self._rich_data)["hostComponentMapRules"]
```

Comment:
is it used somewhere?

Reply:
No, but this field is [described](https://wiki.adsw.io/books/adcm/page/adcm-aio-client#bkmrk-action-attributes%3A-i) in Ation's attributes